### PR TITLE
Change warning message to exception

### LINKED ISSUE #121: Change warning message to exception for markOrder and unmarkOrder

### PR DESCRIPTION
**Pull request by yu-sutong**
close #121

### CODE REVIEW

File: src/main/java/seedu/sellsavvy/logic/commands/ordercommands/UnmarkOrderCommand.java [hunk 1]
```diff
@@ -31,8 +31,7 @@ public class UnmarkOrderCommand extends Command {
 
     public static final String MESSAGE_UNMARK_ORDER_SUCCESS =
             "The order has been reverted to the pending status: %1$s";
-    public static final String MESSAGE_ORDER_ALREADY_UNMARKED_WARNING = "Note: "
-            + "This order is already in the pending status";
+    public static final String MESSAGE_ORDER_ALREADY_UNMARKED = "The order has already been marked as pending.";
```
Comment (FooChao) [inline src/main/java/seedu/sellsavvy/logic/commands/ordercommands/UnmarkOrderCommand.java:34]: this feels weird

File: src/test/java/seedu/sellsavvy/logic/commands/ordercommands/UnmarkOrderCommandTest.java [hunk 2]
```diff
@@ -69,20 +69,10 @@ public void execute_noOrderListDisplayed_throwsCommandException() {
 
     @Test
     public void execute_orderAlreadyUnmarked_warningGiven() {
```
Comment (FooChao) [inline src/test/java/seedu/sellsavvy/logic/commands/ordercommands/UnmarkOrderCommandTest.java:71]: maybe u can change title name?

Comment (yu-sutong) [reply src/test/java/seedu/sellsavvy/logic/commands/ordercommands/UnmarkOrderCommandTest.java:71]: Oops

File: src/main/java/seedu/sellsavvy/logic/commands/ordercommands/UnmarkOrderCommand.java (hunk 1 above)
Comment (kaixuan477) [reply src/main/java/seedu/sellsavvy/logic/commands/ordercommands/UnmarkOrderCommand.java:34]:
```suggestion
    public static final String MESSAGE_ORDER_ALREADY_UNMARKED = "The order is presently not marked as completed";
```
Consider changing "presently" to something else.

File: src/main/java/seedu/sellsavvy/logic/commands/ordercommands/MarkOrderCommand.java [hunk 3]
```diff
@@ -59,13 +58,13 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Order orderToMark = filteredOrderList.get(index.getZeroBased());
+        if (orderToMark.getStatus() == Status.COMPLETED) {
```
Comment (kaixuan477) [inline src/main/java/seedu/sellsavvy/logic/commands/ordercommands/MarkOrderCommand.java:61]: I wonder if we should provide a Status.isCompleted() method to further OOP order status?

**Review by kaixuan477 — COMMENTED** [review]
Good work, small details but looks fine.

File: src/main/java/seedu/sellsavvy/logic/commands/ordercommands/MarkOrderCommand.java (hunk 3 above)
Comment (yu-sutong) [reply src/main/java/seedu/sellsavvy/logic/commands/ordercommands/MarkOrderCommand.java:61]: Sure

**Review by kaixuan477 — APPROVED** [review]

**Pull request merged**